fdbserver/fdbserver.vcxproj

```diff
@@ -167,6 +167,8 @@
     <ActorCompiler Include="workloads\Serializability.actor.cpp" />
     <ActorCompiler Include="workloads\DiskDurability.actor.cpp" />
     <ActorCompiler Include="workloads\SnapTest.actor.cpp" />
+    <ClInclude Include="..\bindings\c\test\mako\zipf.h" />
+    <ClCompile Include="..\bindings\c\test\mako\zipf.c" />
```

Comment:
And thus these should be moved to flow.vcxproj also

Reply:
Will do. Thanks.

Reply:
Done. Thanks.